slicers/fill_layer.go

```diff
@@ -7,16 +7,71 @@ import (
 	"sort"
 )
 
-func CalcFillPlanes(mesh *Mesh, settings Settings) []Plane {
-	minx, maxx := mesh.MinMaxZ(AxisX)
+func CalcFillPlanes0(mesh *Mesh, settings Settings, MyAxis Vector, step float64) []Plane {
```

Comment:
maybe name it with 'common' word and make it hidden ( started with a small letter)